exoflex: Can't resolve '@unimodules/core'

Only happens on web. On mobile, it works fine

```shell
Failed to compile.
/Users/Me/node_modules/expo-font/build/Font.js
Module not found: Can't resolve '@unimodules/core' in '/Users/Me/node_modules/expo-font/build'
```

## Preview
<img width="1280" alt="Screen Shot 2019-09-05 at 11 45 47 AM" src="https://user-images.githubusercontent.com/25707872/64312589-ded39200-cfd2-11e9-91ac-a911dec16011.png">

--- a/packages/exoflex/example/webpack.config.js
+++ b/packages/exoflex/example/webpack.config.js
@@ -25,6 +25,18 @@ module.exports = async function(env, argv) {
   config.resolve.alias['@ptomasroos/react-native-multi-slider'] = path.resolve(
     './node_modules/react-native-multi-slider',
   );
+  config.resolve.alias['@unimodules/core'] = path.resolve(
+    './node_modules/@unimodules/core',
+  );
+  config.resolve.alias['expo-asset'] = path.resolve(
+    './node_modules/expo-asset',
+  );
+  config.resolve.alias['expo-constants'] = path.resolve(
+    './node_modules/expo-constants',
+  );
+  config.resolve.alias['react-native-web/dist/exports'] = path.resolve(
+    './node_modules/react-native-web/dist/exports',
+  );
 
   // Add rule to transform exoflex files before loading it.
   config.module.rules = [

